### PR TITLE
issue 12: Documentation addition for slack_notification support

### PR DESCRIPTION
**Pull request by su7edja**
### Description
Adding documentation for slack_notification support

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues : https://github.com/intuit/Trapheus/issues/12
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

### CODE REVIEW

**Comment by su7edja** [comment]
@namitad could you add "hacktoberfest-accepted" label to the PR if this is acceptable to you?
Thank you.

File: README.md [hunk 1]
```diff
@@ -138,6 +140,7 @@ The following are the parameters for creating the cloudformation template:
 4. `SenderEmail` : [Required] The SES sending email configured in the [Pre-Requisites](#pre-requisites)
 5. `RecipientEmail` : [Required] Comma separated list of recipient email addresses configured in [Pre-Requisites](#pre-requisites).
 6. `UseVPCAndSubnets` : [Optional] Whether to use the vpc and subnets to create a security group and link the security group and vpc to the lambdas. When UseVPCAndSubnets left out (default) or set to 'true', lambdas are connected to a VPC in your account, and by default the function can't access the RDS (or other services) if VPC doesn't provide access (either by routing outbound traffic to a [NAT gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html) in a public subnet, or having a [VPC endpoint](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-endpoints.html), both of which incur cost or require more setup). If set to 'false', the [lambdas will run in a default Lambda owned VPC that has access to RDS (and other AWS services)](https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html#vpc-internet).
+7. `SlackWebhookUrls` : [Optional] Comma separated list of slack webhooks for failure alerts.
```
Comment (namitad) [inline README.md:144]: @su7edja additional info be added in the `setup` steps for sam deploy with slack urls and in `How it works` section specifically the alerting support.

**Review by namitad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (su7edja) [reply README.md:144]: Thanks for the review @namitad 
I added the steps you mentioned above. Could you add "hacktoberfest-accepted" label to the PR if this is acceptable?

**Comment by su7edja** [comment]
@namitad let me know if there is anything else

File: README.md [hunk 2]
```diff
@@ -61,6 +61,8 @@ Trapheus can be scheduled using a cloudwatch alarm rule or can be run using any
 │   ├── restore
 │   │   ├── cluster_restore_function.py          <-- Python Lambda code for retoring a clusterised database.
 │   │   └── restore_function.py                  <-- Python Lambda code for restoring a non-clusterised RDS instance
+│   ├── slackNotification
+│   │   └── slack_notification.py                <-- Python Lambda code for sending out a failure alert to configured webhook(s) on Slack.
 │   └── snapshot
```
Comment (namitad) [inline README.md:66]: Nit: do add the details of the test file for slack_notification.py

Comment (su7edja) [reply README.md:66]: @namitad added

**Comment by su7edja** [comment]
@namitad could you add the hacktoberfest label to this PR?

**Review by namitad — APPROVED** [review]
LGTM

**Review by stationeros — APPROVED** [review]

**Pull request merged**